src/components/Menu/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "GothamRnd-Book";
+  src:
+    local("GothamRnd-Book"),
+    url("./GothamRnd-Book.otf");
+}
+
+:root {
+  --mColor: #0081c9;
+  --mFont: "GothamRnd-Book";
+}
+
+.menu {
+  background-color: var(--mColor);
+  font-family: var(--mFont);
+  color: white;
+  font-size: 20px;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  overflow: hidden;
+  background-color: var(--mColor);
+}
+
+li {
+  float: left;
```

Comment:
be aware aboute the float !!!!

Reply:
aboute? Haceeene, voyons :D

Reply:
@Agent-Roman-Off Bashing/Trolling is allowed (with a modicum of moderation, of course), **aber Französisch ist verbôten ! ರ╭╮ರ**